docs/conf.py

```diff
@@ -67,3 +70,14 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 # html_static_path = ["_static"]
+
+
+# Auto-generate API documentation
+# Source: https://github.com/readthedocs/readthedocs.org/issues/1139
+def run_apidoc(_):
```

Comment:
@vringar, do you know how I should type annotate the `_` parameter? Or should we just tell `mypy` to ignore function definitions in this file?

Reply:
You could annotate it with Any
```suggestion
def run_apidoc(_: Any) -> None:
```
so that mypy stops complaining.
But I think telling mypy to not enforce type annotations in the `docs` folder would be better.